### ISSUE TITLE
Problem - No sounds - Asus C100P

### ISSUE THREAD

**Issue by rshalaev** [austin987/crawfishos#10]
I installed Crawfish OS on 2 Asus C100P Flip Chromebooks - one with Gnome, another one with XFCE4. Both have an issue that they don't have sound - tested with VLC and with Firefox. Would much appreciate help with troubleshooting. Thank you in advance!

**Comment by nwg-piotr** [comment]
Same on Asus C201PA.

**Comment by rshalaev** [comment]
> Same on Asus C201PA.

Did you try upstream [PrawnOS](https://github.com/SolidHal/PrawnOS)? - will be my next step to try it out there

**Comment by nwg-piotr** [comment]
It won't work w/ built-in wi-fi chip, AFAIK.

**Comment by rshalaev** [comment]
> It won't work w/ built-in wi-fi chip, AFAIK.

That's true. However sound I think is a separate issue. Reasoning to try out PrawnOS is 2-folds:
1) It might actually have a working sound - in that case I'd then focus on whatever differences were introduced between PrawnOS and CrawfishOS to troubleshoot
2) If PrawnOS sound is also broken - it would be best to file a bug report in their GitHub based directly on their distro, rather than a derivative

**Comment by nwg-piotr** [comment]
Once I tried Cadmium OS. It was not really simple to set up sound there, but finally it worked for me. Unfortunately later builds stopped working on C201 PA, and I gave up. Dunno if it's fixed now.

**Comment by rshalaev** [comment]
> Cadmium OS

Thanks for the heads up!

**Comment by austin987** [comment]
The main difference is that CrawfishOS _enables_ the builtin wifi chip, so yeah, that could definitely be it.

I don't have access to a C100P, so I don't have much more to help.

**Comment by nwg-piotr** [comment]
Internal wifi worked for me on Cadmium, but I had other issues. So my Asus is back on Crawfish.

**Comment by rshalaev** [comment]
I tested [PrawnOS Shiba v1.3.1 - Usb boot fix](https://github.com/SolidHal/PrawnOS/releases/tag/v1.3.1) with Asus C100P - sound is working

Next step - perhaps I'll do a diff on PrawnOS to CrawfishOS  source code and see if anything apparent stands out. As [austin987](https://github.com/austin987) mentioned - perhaps the act of enabling wifi chip disabled the sound, but hopefully it's something else and can be easily resolved.

**Comment by Bio7269** [comment]
I had this issue both on PrawnOS and CrawfishOS on an Asus C201PA with libreboot.
After a fresh install(GNOME DE), everything except sound would function.
I was able to get audio working after multiple attempts in the gnome sound settings.

The Fix for me was to insert headphones into the 3.5mm audio jack and when removed, internal audio started working.
Since then, internal audio works persistently, even after a reboot.

Specifically after plugging in the headphones, I tested the audio for functionality on the headphones in the sound settings..
After confirming audio was working on the headphones, I unplugged the 3.5mm audio jack, the default output device switched back to internal(Built-In) audio.

This was the only issue I encountered, but I'm glad I was able to get it fixed. I hope this information can help those experiencing the same issue.

**Comment by nwg-piotr** [comment]
> The Fix for me was to insert headphones into the 3.5mm audio jack and when removed, internal audio started working.

~~This helped when I fist installed Crawfish, but didn't on the current install.~~ No idea why. But I have the sound in headphones.

[edit] Honestly, I don't really remember if it helped on Crawfish or Cadmium, sorry.

**Comment by cracket** [comment]
Big challenge for C100P audio are dozens of mixer setting for audio - if you ever came across audio problems - make sure you are able to provide and reproduce alsa mixer settings
